src/components/Canvas/DataLayer.tsx

```diff
@@ -259,6 +261,14 @@ export default class DataLayer extends BaseLayer {
     }
   }
 
+  setGridSquareLength(length: number) {
```

Comment:
```suggestion
  set gridSquareLength(length: number) {
```
just a little question, for the getters and setters why are we not using these methods? is it just a coding style?

Reply:
Good point. I was not familiar with the getter/setter of the javascript language. For now, I will just stick with the setXXX function naming. There would be too much code to rename if I decided to rename this particular function to a setter. But thank you for the review!